suite_cam6.xml

```diff
@@ -0,0 +1,363 @@
+      <scheme>calc_atm_density</scheme>               <!-- Calculate atmospheric density -->
+      <subcycle loop="dep_velocity_idx_34">           <!-- loop over deposition velocity (vlc) array index values 3 and 4 -->
+        <scheme>modal_aero_depvel_part</scheme>       <!-- Calculate dry deposition velocity for cloud droplets -->
+      </subcycle>
+      <subcycle loop="aerosol_dist_modes">            <!-- Loop over aerosol size distribution modes, CAM6 var = 'ntot_amode' -->
+        <subcycle loop="aerosol_phase">               <!-- Loop over aerosol phase types (interstitial vs cloud-borne), CAM6 var = 'lphase' -->
+          <scheme>calc_aero_vars</scheme>             <!-- Calculate aerosol mean wet radius and other properties -->
+          <subcycle loop="dep_velocity_idx_12" >      <!-- loop over deposition velocity (vlc) array index values 1 and 2 -->
+            <scheme>modal_aero_depvel_part</scheme>   <!-- Calculate dry deposition velocity for interstial aerosols -->
+          </subcycle>
+          <scheme>aero_tracer_indx</scheme>           <!-- Determine aerosol tracer index (managed by registry/framework?) -->
+          <scheme>depvel_m_to_pa</scheme>             <!-- Convert the deposition velocity from m/s to Pa/s, and write to history file (maanged by registry/framework?) -->
+          <scheme>dust_sediment_tend</scheme>         <!-- Calculate host model tendencies and surface fluxes from dry deposition -->
+          <scheme>drydep_diag_output</scheme>         <!-- Calculate dry deposition diagnostic variables, and output them to history file -->
+        </subcycle>
+      </subcycle>
+      <scheme>set_srf_drydep</scheme>                 <!-- Add dry deposition surface fluxes to host model, if aerodep_flx_prescribed is False -->
+      <!-- **************************** -->
+      <scheme>charge_balance</scheme>                 <!-- Enforce charge nuetrality, if chemistry simulates electrons -->
+      <!-- Gravit wave tendencies -->
+      <!-- **************************** -->
+      <!-- This interface depends heavily
+           on namelist inputs, with entire
+           blocks of code being used (or not)
+           depending on said namelist choices.
+           However, the general structure of these
+           code blocks is more or less the same,
+           so a "generic" scheme set is included
+           here, which will need to be expanded
+           out once "if-statement" like dependencies
+           can be added to suite definition files
+           like this one -->
+      <scheme>set_dry_to_wet</scheme>                  <!-- Convert mixing ratios from dry air to air + water vapor -->
+      <scheme>gw_prof</scheme>                         <!-- Calculate layer interface densities and Brunt-Viasala frequencies (combine with other schemes?) -->
+      <!-- Generic gravity wave scheme order  -->
+      <!-- +++++++++++++++++++++++++ -->
```

Comment:
Isn't Julio working on a CCPP version of gravity wave?  If so, can its suite be put here?

Reply:
It is way too soon for that. When he is ready, his version could replace the current GW code in our suite.